### PR TITLE
[OSDOCS-5069]: VMware OoT provider is GA (rel notes)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.13

Issue:
[OCPCLOUD-1129](https://issues.redhat.com//browse/OCPCLOUD-1129) | [OSDOCS-5070](https://issues.redhat.com//browse/OSDOCS-5070)

Link to docs preview:
- [Cloud controller managers for additional cloud providers](https://55358--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-cluster-cloud-controller-manager-operator) in _Notable technical changes_
- [Technology Preview tracker](https://55358--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#machine-management-technology-preview-features) table

QE review:
- [x] QE has approved this change.

Additional information:
- Left some text commented out that I plan to use for another related feature when I get to that (Nutanix CCMs)
- Also fixed a few things in the rel notes

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@lobziik and this one as well :bow:

**Comment by jeana-redhat** [comment]
@huali9 this is ready for you too, PTAL :)

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -111,6 +111,23 @@ This release adds improvements related to the following components and concepts.
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-13-cluster-cloud-controller-manager-operator"]
+==== Cloud controller managers for additional cloud providers
+
+The Kubernetes community plans to deprecate the Kubernetes controller manager in favor of using cloud controller managers to interact with underlying cloud platforms. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms.
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-13-release-notes.adoc]: This is a bit wordy, but is it more accurate?
https://github.com/openshift/openshift-docs/pull/55358/files#diff-5a1bef860ba4f103b12cdc2db661140fa691167a8b50abea180008754482951cR137

**Comment by huali9** [comment]
LGTM

**Comment by JoelSpeed** [comment]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -149,7 +166,7 @@ In the following tables, features are marked with the following statuses:
 .Operator deprecated and removed tracker
 [cols="4,1,1,1",options="header"]
 |====
-|Feature |4.11 |4.12 | 4.13
+|Feature |4.11 |4.12 |4.13
```
Comment (JoeAldinger) [inline release_notes/ocp-4-13-release-notes.adoc:169]: Thank you!

**Review by JoeAldinger — COMMENTED** [review]
/lgtm
Thank you for the catch before the chaos ensued.
